### ISSUE TITLE
[Sidenav] Sidenav does not complete state changes when animations are disabled.

### ISSUE THREAD

**Issue by josephperrott** [angular/material2#7053]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Sidenav makes state changes without animations.

#### What is the current behavior?
Sidenav does not complete state changes and some animations still occur.

#### What are the steps to reproduce?
Add `[@.disabled]` to the element containing a sidenav container and attempt to use sidenav.

Providing a Plunker (or similar) is the *best* way to get the team to see your issue.
https://plnkr.co/edit/v29ARj2yt3vQbeBsEqVf?p=preview

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
@angular/angular 4.3.6
@angular/material 2.0.0-beta.10

At least chrome and firefox

#### Is there anything else we should know?
The average set of human lungs weigh 840g (1.85lbs) [source](https://www.ncbi.nlm.nih.gov/pubmed/22182984)

**Comment by mmalerba** [comment]
updated plunker: https://plnkr.co/edit/k4PlA4rL7G2nqGe0Fxwk?p=preview

**Comment by viksakshum** [comment]
yes this is a big problem for us as well. please advise a solution

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
